index.html

```diff
@@ -82,16 +82,16 @@ <h4>Skills</h4>
                 <li><i class="fa-solid fa-m"></i></li>
             </ul>

```

Comment:
- [ ]  I recommend adding links 44 - 84 in its own section. This will improve your code readablity and will make your project mare sementic.  The important of HTML5’s semantic elements help structure the code we create, making it more readable and easier to maintain. They help us think about the structure of our dynamic data, and to choose titles’ hierarchy properly.